pkg/radar/threat.go

```diff
@@ -0,0 +1,49 @@
+package radar
+
+import (
+	"github.com/dharmab/skyeye/pkg/bearings"
+	"github.com/dharmab/skyeye/pkg/brevity"
+	"github.com/dharmab/skyeye/pkg/coalitions"
+	"github.com/martinlindhe/unit"
+	"github.com/paulmach/orb/geo"
+)
+
+func (s *scope) Threats(coalition coalitions.Coalition) map[uint32][]brevity.Group {
+	threats := make(map[uint32][]brevity.Group)
+	itr := s.contacts.itr()
+	for itr.next() {
+		trackfile := itr.value()
+
+		if trackfile.Contact.Coalition == coalition {
+			continue
+		}
+
+		if !isValidTrack(trackfile) {
+			continue
+		}
+
+		groups := s.findNearbyGroups(trackfile.LastKnown().Point, 0, 100000, 100*unit.NauticalMile, coalition, brevity.FixedWing)
```

Comment:
constants?

Reply:
I made  some earlier but forgot their names so just typed beeg numbers in to make the code work lol